selfdrive/camerad/cameras/real_debayer.cl

```diff
@@ -13,30 +13,33 @@ const __constant half3 color_correction_2 = (half3)(-0.25277411, -0.05627105, 1.
 // tone mapping params
 const half gamma_k = 0.75;
 const half gamma_b = 0.125;
-const half mp_default = 0.01; // ideally midpoint should be adaptive
+const half mp = 0.01; // ideally midpoint should be adaptive
+const half rk = 9 - 100*mp;
 
-half gamma_apply(half x, half mp) {
+inline half3 gamma_apply(half3 x) {
   // poly approximation for s curve
-  half rk = 9 - 100*mp;
-  if (x > mp) {
-    return (rk * (x-mp) * (1-(gamma_k*mp+gamma_b)) * (1+1/(rk*(1-mp))) / (1+rk*(x-mp))) + gamma_k*mp + gamma_b;
-  } else if (x < mp) {
-    return (rk * (x-mp) * (gamma_k*mp+gamma_b) * (1+1/(rk*mp)) / (1-rk*(x-mp))) + gamma_k*mp + gamma_b;
-  } else {
-    return x;
-  }
+  return (x > mp) ?
```

Comment:
This is now comparing a half3 to a scalar? Something like `select` sounds more appropriate here.

Reply:
@jwooning pointed out that the ternary operator with a vector is special in OpenCL and is the same as select: https://www.khronos.org/registry/OpenCL/sdk/2.2/docs/man/html/selectionOperator.html

Reply:
Yea I just copied that from Joost's PR